recipes/gromacs/2020/meta.yaml

```diff
@@ -23,7 +24,6 @@ requirements:
     - cmake
     - pocl
   host:
-    - openmp # [linux]
```

Comment:
Why this change? OpenMP is arguably  more useful than MPI or OpenCL to bioconda users

Reply:
Thanks for spotting this, it is just a mistake. I was (to quickly) removing the changes brought in to enable MPI whilst trying to keep the non-mpi changes and deleted the wrong lines.

I thought the build time seemed odd, but when Stian re-ran the build it was fine.  I wonder what has caused the time out.  I shall try and look at this today, feel free to revert and I will do a new PR later.